source/collections/FloatVectorList.ooc

```diff
@@ -30,6 +35,19 @@ FloatVector: class extends VectorList<Float> {
 		result count = this count
 		result
 	}
+
+	discreteFourierTransform: static func (input: This) -> This {
+		result := This new(input count)
+		for (i in 0..(input count)) {
+			tempVariable:= FloatComplex new(0,0)
```

Comment:
`tempVariable` is not a very descriptive name. Call it `accumulator`, `sum` or something.